src/preact/context.tsx

```diff
@@ -16,10 +16,10 @@ export interface AmpContext {
 }
 
 export interface ProviderProps {
-  renderable: boolean | undefined;
-  playable: boolean | undefined;
-  loading: string | undefined;
-  notify?: () => {} | undefined;
+  renderable?: boolean;
+  playable?: boolean;
+  loading?: string;
+  notify?: () => {};
```

Comment:
Is this a function that returns an object?

Reply:
thunk that returns void

Reply:
Then let's make it `() => void`. Currently I think it may be typed as the literal value of that no-op function.